test/react.test.js

```diff
@@ -29,8 +29,7 @@ describe('styled', () => {
   })
 
   test('basic render with object as style', () => {
-    const fontSize = 20
-    const H1 = styled.h1({ fontSize })
+    const H1 = styled.h1({ fontSize: 20 })
```

Comment:
No, array values are not the issue at all, just happened to be the last thing I fixed. 

I build an object from AST here: https://github.com/tkh44/emotion/pull/101/files#diff-bb35c33015964b91f5c83deb2a33a814R128, so need an actual scalar value. I think it should be easier to resolve the literal value if defined in the same module like done in that test. But imported ones would be little harder since we'd have to resolve imported val via reading/parsing modules.

I will look at resolving literals defined in same module later today.